locust/configuration.py

```diff
@@ -89,6 +89,39 @@ def update_json_config(self, json_added, json_path, options, list_column, config
             data = ClientConfiguration.update_json(data, ClientConfiguration.get_path(match), json_final)
         
         return make_response(json.dumps({'success':True, 'data':json.dumps(data, indent=4)}))
+
+    def add_new_key(self, parent_path, new_key_name, new_key_type, config_text):
+        data = literal_eval(config_text)
+        data_target = data
+        temppath = parent_path.split('.')
+        child_split = new_key_name.split('.')
+
+        counts = {}
+        branch = counts
+        for part in child_split[1:]:
+            if part == child_split[-1]:
```

Comment:
what if there is key with the same name in different level?
this is just for example $.production.accounts.address.city.address